modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -2,11 +2,17 @@
 
 .ratings {
 	.btn {
+		align-items: center;
+		display: inline-flex;
```

Comment:
I have noticed that the thumbs-up and thumbs-down are utilizing different css. I've seen this implementation be applied to only one of them.

Reply:
This also applies to `type="like"`. There's no way to change it via UI in Portal (not that I could find). You need to manually change/create them it in the code and deploy.